### ISSUE TITLE
Max size for decryption request ciphertext

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#3176]
Let's put a cap on the max size that Ursulas would accept as a decryption request.
Include an error message that suggests to try hybrid encryption.

**Comment by jdbertron** [comment]
What's hybrid encryption ? Some kind of encryption of a symmetric key ?

**Comment by cygnusv** [comment]
Correct, that's exactly it. In fact, the Ferveo TDec cryptosystem we use already does that, but it doesn't expose it explicitly.

**Comment by cygnusv** [comment]
Very related: https://github.com/nucypher/ferveo/issues/147

**Comment by derekpierre** [comment]
Encapsulation work being done via:
- https://github.com/nucypher/ferveo/pull/149
- https://github.com/nucypher/ferveo/issues/155
- https://github.com/nucypher/ferveo/pull/156
- https://github.com/nucypher/nucypher-core/pull/74
- https://github.com/nucypher/nucypher/pull/3194
